t/test01.t

```diff
@@ -4,6 +4,7 @@ use 5.14.2;
 
 use Test::More;    # see done_testing()
 use Zonemaster;
+use JSON;
```

Comment:
If you use JSON it will automatically use JSON::PP since both are core packages. But it will also automatically pick up JSON::XS if it is installed, which I would very much prefer. It increases performance.